raiden_installer/utils.py

```diff
@@ -31,9 +31,9 @@ def get_contract_address(chain_id, contract_name):
         network_contracts = get_contracts_deployment_info(chain_id)
         assert network_contracts
         return network_contracts["contracts"][contract_name]["address"]
-    except (TypeError, AssertionError) as exc:
+    except (TypeError, AssertionError, KeyError) as exc:
         log.warn(str(exc))
-        return "0x0"
+        raise ValueError(f"{contract_name} does not exist on chain id {chain_id}")
```

Comment:
Id suggest to use `raise ValueError(...) from exc` to clarify the cause of the exception in the traceback.

Reply:
I didn't know about that syntax. I will use it. Thank you!